src/studentView.pug

```diff
@@ -0,0 +1,53 @@
+#studentView(hidden='')
+    header
+        #date
+        #account-icon
+            img(src='./gui/img/User.png' alt='User Icon')
+            span#account-username-display Welcome, User
+            #logout
+                span Logout
+    .node-search(ondragover='allowDroppingNode(event)' ondrop='deleteNode(event)')
```

Comment:
The node search will appear in multiple views (i.e. student and teacher, plus maybe different places in the student view like practice vs assignments). It should have a template